### ISSUE TITLE
CLI ERROR: it can not run under Mac OS Mojave 10.14.6

### ISSUE THREAD

**Issue by ZhiweiYAN** [omniedgeio/omniedge#15]
The Error Info is like this:

USR520:omniedgedir foo$ ./omniedge login -u email 
dyld: Symbol not found: ___darwin_check_fd_set_overflow
  Referenced from: /Users/foo/Desktop/temp/omniedgedir/./omniedge (which was built for Mac OS X 11.0)
  Expected in: /usr/lib/libSystem.B.dylib
 in /Users/foo/Desktop/temp/omniedgedir/./omniedge
Abort trap: 6

**Comment by yongqianme** [comment]
Yes. Because we compile and release the omniedge macOS Cli version by GitHub workflow.  The lowest version of  macOS runner is macos-10.15. https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners

However, you can compile by yourself if you really need run it for version below 10.15. 

```
brew install autoconf automake libtool
git clone https://github.com/omniedgeio/omniedge-cli
cd omniedge-cli
go mod download
go generate
BUILD_ENV=prod make build-darwin

```

**Comment by ZhiweiYAN** [comment]
Many thanks Yong.

There are still two problems you might know,  although I have solved the first one. 

First One: It happened during the process of compiling. 
---------------------
$ BUILD_ENV=prod make build-darwin
go generate ./...
Cloning into 'n2n'...
.... ...
Switched to a new branch '2.6-stable-omni'

Now run ./configure
checking for gcc... gcc
... ...
config.status: creating config.h
sh: ../make: Permission denied
internal/edge.go:3: running "sh": exit status 126
make: *** [generate] Error 1
-----------------------
After I give the script of 'make' the execute attribute, it seems OK.  

However,  I am struggling with the second one until now. 
The second one happened after I login successful.  
And then I run the cmd 'omniedge join', and then select the virtual network.  
The error info is following: 

-----------------------------
./omniedge join
INFO[0000] You are in mode: prod                        
INFO[2022-07-30T20:16:38] List Virtual Network response                
🌶 My Omni Network
You choose number 1: My Omni Network
INFO[2022-07-30T20:16:49] Success to join virtual network              
INFO[2022-07-30T20:16:49] Start to connect omniedge                    
2022-07-30T20:16:49 [edge_utils.c:1943] INFOAdding supernode[0] = prod-useast.edgecomputing.network:443
2022-07-30T20:16:49 [tuntap_osx.c:49] ERRORUnable to open any tap devices /dev/tap0 through /dev/tap254. Is this user properly authorized to access those descriptors?
2022-07-30T20:16:49 [tuntap_osx.c:50] ERRORPlease read https://github.com/ntop/n2n/blob/dev/doc/n2n_on_MacOS.txt
ERROR[2022-07-30T20:16:49] fail to open TUN/TAP device    
------------------------

BTW: 
- The webpage of 'https://github.com/ntop/n2n/blob/dev/doc/n2n_on_MacOS.txt' does not exists. 
- I have installed the tap/tun driver according the instructions from https://omniedge.io/docs/article/install/macos. 

Thanks in advance.
